clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -3912,6 +4012,12 @@ void SYCLIntegrationHeader::addSpecConstant(StringRef IDName, QualType IDType) {
   SpecConsts.emplace_back(std::make_pair(IDType, IDName.str()));
 }
 
+void SYCLIntegrationHeader::setCallsThisItem(bool B) {
+  auto *K = getCurKernelDesc();
```

Comment:
Please don't use auto here, because it is not obvious what is the type of return value of `getCurKernelDesc()` function.

Reply:
OK